.github/workflows/publish.yml

```diff
@@ -1,11 +1,16 @@
 name: "Publish"
 on:
-  push:
+  pull_request:
     branches:
-      - master
+      - release/**
+      - hotfix/**
+    types: [closed]
+
 
 jobs:
-  Pipeline:
+  build:
+    if: github.event_name == 'pull_request' && github.event.action == 'closed' && github.event.pull_request.merged == true
```

Comment:
what is this?

Reply:
Conditional to prevent a job from running. You can read about this [here](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idif).